webapp/frontend/app/templates/lambda-app.hbs

```diff
@@ -9,6 +9,22 @@
 
 <!-- Main content -->
 <section class="content">
+
+  {{#if request}}
```

Comment:
Maybe it is better to move the message box on top of the second table, in order to be closer to the buttons.